File: public/docs/_examples/component-styles/ts/app/hero-team.component.ts
import {Component, Input} from 'angular2/core';
import {Hero} from './hero';

// #docregion stylelink
@Component({
  selector: 'hero-team',
  template: `
    <link rel="stylesheet" href="app/hero-team.component.css">
    <h3>Team</h3>
    <ul>
      <li *ngFor="#member of hero.team">
        {{member}}
      </li>
    </ul>
  `
})
export class HeroTeamComponent {
// #enddocregion stylelink

  @Input() hero:Hero;
}

File: public/docs/_examples/component-styles/dart/lib/hero_team_component.dart
import 'package:angular2/core.dart';
import 'hero.dart';

// #docregion stylelink
@Component(
    selector: 'hero-team',
    template: '''
      <link rel="stylesheet" href="hero_team_component.css">
      <h3>Team</h3>
      <ul>
        <li *ngFor="let member of hero.team">
          {{member}}
        </li>
      </ul>''')
class HeroTeamComponent {
  @Input() Hero hero;
}
